### PR TITLE
Add more servered characters (<, >, ', ", `, ' ') and test cases

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add more servered characters (<, >, ', ", `, SPACE) and test cases

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/241

### CODE REVIEW

File: unittest/state-util.test.js [hunk 1]
```diff
@@ -288,13 +309,31 @@ describe("state-util", () => {
...
+      expect(isValidStateLabel('%')).to.equal(true);
+      expect(isValidStateLabel('^')).to.equal(true);
+      expect(isValidStateLabel('&')).to.equal(true);
+      expect(isValidStateLabel('-')).to.equal(true);
+      expect(isValidStateLabel('_')).to.equal(true);
+      expect(isValidStateLabel('=')).to.equal(true);
+      expect(isValidStateLabel('+')).to.equal(true);
+      expect(isValidStateLabel('|')).to.equal(true);
+      expect(isValidStateLabel(';')).to.equal(true);
+      expect(isValidStateLabel('|')).to.equal(true);
```
Comment (liayoo) [inline unittest/state-util.test.js:334]: Duplicate? (L332)

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks!

File: unittest/state-util.test.js (hunk 1 above)
Comment (platfowner) [reply unittest/state-util.test.js]: Fixed. Thanks!

**Comment by platfowner** [comment]
Thanks for fast review! Now merging.. @liayoo

**Pull request merged**